x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -313,8 +360,46 @@ export class ESGeoGridSource extends AbstractESAggSource {
     };
   }
 
+  getLayerName() {
+    return MVT_SOURCE_LAYER_NAME;
+  }
+
+  async getUrlTemplateWithMeta(searchFilters) {
+    const indexPattern = await this.getIndexPattern();
+    const searchSource = await this.makeSearchSource(searchFilters, 0);
+
+    this._addNonCompositeAggregationsToSearchSource(searchSource, {
+      indexPattern,
+      precision: -1, // This needs to be set server-side
```

Comment:
Instead of setting to an invalid value, can precision just be left unset in the request body?